README.md

```diff
@@ -9,7 +9,7 @@ Simple low-overhead circuit breaker library.
 ## Usage
 
 ```go
-h := hoglet.NewCircuit(
+h, _ := hoglet.NewCircuit(
```

Comment:
nit: we probably shouldn't give people the impression to ignore these errors 😬 
```suggestion
h, err := hoglet.NewCircuit(
```

Reply:
but then should we also add the actual error-handling to the example? Or just with `/* ... error handling ... */` markers, so that copy-pasting won't compile? :see_no_evil:

Reply:
I think markers are ok

Reply:
done, PTAL